src/main/scala/li/cil/oc/common/init/Items.scala

```diff
@@ -540,6 +540,8 @@ object Items extends ItemAPI {
     registerItem(new item.DiamondChip(multi), Constants.ItemName.DiamondChip)
     Recipes.addSubItem(new item.UpgradeMF(multi), Constants.ItemName.MFU, "oc:mfu")
 
+    registerItem(new item.ComponentBus(multi, Tier.Four), Constants.ItemName.ComponentBusCreative)
```

Comment:
add comment for version here, // 1.7.2
as was done above previously

Reply:
i'll add this comment later

Reply:
Thanks. I wasn't sure if that should wait; I'm not used to adding version numbers before hitting a milestone.